ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/DbChangelog.hs

```diff
@@ -70,30 +70,28 @@ import qualified Ouroboros.Network.AnchoredSeq as AS
 -- We illustrate its contents below, where @k = 3@ (for a state @Li@, the
 -- corresponding set of differences is @Di@):
 --
--- > stateAnchor | diskAnchor | states                     | tableDiffs
--- > --------------------------------------------------------------------------
--- >      0      |      0     | [ L0 ]                     | [ ]
--- >      0      |      0     | [ L0, L1 ]                 | [ D1 ]
--- >      0      |      0     | [ L0, L1, L2 ]             | [ D1, D2 ]
--- >      0      |      0     | [ L0, L1, L2, L3 ]         | [ D1, D2, D3 ]
--- >      1      |      0     | [ L0, L1, L2, L3, L4 ]     | [ D1, D2, D3, D4 ]
--- >      2      |      0     | [ L0, L1, L2, L3, L4, L5 ] | [ D1, D2, D3, D4, D5 ]    (*)
--- >      2      |      2     | [ L2, L3, L4, L5 ]         | [ D3, D4, D5 ]   -- flush (**)
--- >      3      |      2     | [ L2, L3, L4, L5, L6 ]     | [ D3, D4, D5, D6 ]
+-- >  lastFlushed | states                         | tableDiffs
+-- > -------------------------------------------------------------
+-- >       0      | [ L0 ]                         | [ ]
+-- >       0      | [ L0, L1 ]                     | [ D1 ]
+-- >       0      | [ L0, L1, L2 ]                 | [ D1, D2 ]
+-- >       0      | [ L0, L1, L2, L3 ]             | [ D1, D2, D3 ]
+-- >       0      | [     L1, L2, L3, L4 ]         | [ D1, D2, D3, D4 ]
+-- >       0      | [         L2, L3, L4, L5 ]     | [ D1, D2, D3, D4, D5 ]    (*)
+-- >       2      | [         L2, L3, L4, L5 ]     | [         D3, D4, D5 ]   -- flush (**)
+-- >       2      | [             L3, L4, L5, L6 ] | [         D3, D4, D5, D6 ]
```

Comment:
I think it's still useful to have `stateAnchor` here (but possibly remaned), to illustrate how the disk anchor only changes when flushed, and the `stateAnchor` is the anchor of the volatile states that changes on every block we apply and push

Reply:
The comment below should also be updated if we bring back `stateAnchor`